### PR TITLE
Add missing type to build-administrator-role-relationships step

### PR DESCRIPTION
**Pull request by charlieduong94**
oops forgot about this. It's pretty easy to get a relationship's type by just building it and seeing what happens, but it might be helpful to add documentation on how types are generated.

### CODE REVIEW

File: src/steps/build-administrator-role-relationships/index.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { STEP_ID as ROLE_STEP, ROLE_TYPE } from '../fetch-administrator-roles';
 const step: IntegrationStep = {
   id: 'build-administrator-role-relationships',
   name: 'Build administrator role relationships',
-  types: [],
+  types: ['trend_micro_administrator_assigned_role'],
```
Comment (aiwilliams) [inline src/steps/build-administrator-role-relationships/index.ts:16]: How did you discover this was missing? Is there an opportunity to make the SDK warn about this when the collect is run?

Comment (charlieduong94) [reply src/steps/build-administrator-role-relationships/index.ts:16]: Just noticed it when I started building the next set of relationships. I definitely think the sdk should be tweaked warn when generated `_types` do not match the declared types.

Comment (charlieduong94) [reply src/steps/build-administrator-role-relationships/index.ts:16]: I'll create an issue

Comment (charlieduong94) [reply src/steps/build-administrator-role-relationships/index.ts:16]: https://github.com/JupiterOne/integration-sdk/issues/74

**Pull request merged**